TombEngine/Renderer/RendererDrawEffect.cpp

```diff
@@ -497,10 +497,10 @@ namespace TEN::Renderer
 				}
 
 				// Disallow sprites out of bounds.
-				int spriteIndex = std::clamp((int)particle.spriteIndex, 0, (int)_sprites.size());
+				int spriteIndex = std::clamp((int)particle.SpriteID, 0, (int)_sprites.size());
```

Comment:
I am afraid this clamp will not work properly anymore, because `spriteIndex` was converted to `SpriteID` and `SpriteSeqID`.